build.sh

```diff
@@ -49,7 +49,7 @@ if [ "${1}x" == "x" ] || [ "${1}" == "--help" ] || [ "${1}" == "-h" ]; then
   echo "  DOCKERFILE  The name of Dockerfile to use."
   echo "              Default: Dockerfile"
   echo "  DOCKER_FROM The base image to use."
-  echo "              Default: 'python:3.9-alpine'"
+  echo "              Default: 'alpine:edge'"
```

Comment:
Unless there is a specific reason we have to use `alpine:edge` and since `edge` is – at least currently – a downgrade to Python 3.8 anyway, I suggest to stick to `alpine:3.12` for now. We will still get at patch-level updates for Python and Unit, but won't have to 'suddenly' experience major-level updates of packages that potentially break our scripts or don't work with upstream Netbox yet.

```suggestion
  echo "              Default: 'alpine:3.12'"
```

Reply:
The unit version in 3.12 is really old, that's why I'm using egde here. As soon as 3.13 is released we should switch to that.